app/controllers/concerns/cached_columns.rb

```diff
@@ -0,0 +1,21 @@
+module CachedColumns
```

Comment:
I hear you, but don't love it. I think "Field" something somewhere should be part of this concern. Right now "CachedColumns" could mean a lot of things.

Is it worth adding a description as well?

Thinking about the behaviour / concern, it seems this provides some sort of functionality for actions with record tables?

Reply:
`CachedColumns` -> `DataTableFieldHeaderCalculatorExtrodinare`

i kid...

`CachedColumns` -> `DynamicFieldNamesCacher`